### ISSUE TITLE
Posts by Webhook do not get the bot to work.

### ISSUE THREAD

**Issue by Hammer2000** [ArtemGr/Sidekick#176]
I created a webhook to post to my discord server, and was hoping a simple content of something like '/r 1d6+10' would get the bot to respond, but it appears the bot only reacts to member posts. I can always make my webhook do the rolls itself but would've been a neat feature if it worked.

**Comment by ArtemGr** [comment]
Yes, the bot skips messages from other bots. I doubt that simply enabling the processing of other bot messages is a right next move there, though.

**Comment by Hammer2000** [comment]
I agree, it could be a bit much, especially if it led to some unintended bot to bot loop. I'll just see about programming in the rolls to my webhook.

**Comment by ArtemGr** [comment]
It should be NP to whitelist your bot's user ID, if you want.

**Comment by Hammer2000** [comment]
Only been working with webhooks a little bit, is that some setting I can apply to my instance of Sidekick bot or something you'd have to add to the bot itself? If I'm using a webhook, would that mean I should use the webhook ID?

**Comment by ArtemGr** [comment]
It is something I'll hardcode in the bot (the second option).

**Comment by sebastian-meyer** [comment]
Can you hardcode some popular bots like [MEE6](https://mee6.xyz/) into Sidekick? That way one could use MEE6's custom command feature to create shortcut commands for often needed dice rolls (or other fancy features).

**Comment by ArtemGr** [comment]
Picking the bots to listen to might be an option for #160, I think. Maybe we'll call the engine asking it for some metadata and there's be a list of bots it listens to.

**Comment by mbfrydman** [comment]
Instead of hardcoding maybe we can message the bot to whitelist?
Or we could add a string at the end of bots names that want to be seen by sidekick? Something like mybot-sdkk

Could be simple.

**Comment by ArtemGr** [comment]
Yeah, this is certainly an option!
BTW, any tidbits on your UI? I'm working on a kind of mobile UI myself, though things get delayed now and then.

**Comment by mbfrydman** [comment]
It's done and working, currently only supports some of your features.
I need some more testing and then I could share if there is interest

**Comment by ArtemGr** [comment]
Sure, paint me interested!
If you'll send me the user ID and the name of the bot then I'll experimentally hardcode them.

**Comment by mbfrydman** [comment]
OK let's try with : 702193297375887492

**Comment by mbfrydman** [comment]
do you have a discord server for sidekick we can chat on?

**Comment by ArtemGr** [comment]
yep, https://discord.gg/bNunVhD. i've added 702193297375887492 two days ago but didn't restart the shards yet. restarting them now
